### PR TITLE
[geometry] Define better how to transform a point with a matrix

### LINKED ISSUE #183: [geometry] DOMPointReadonly  matrixTransform() is poorly specified
https://drafts.fxtf.org/geometry/#dom-dompointreadonly-matrixtransform

Step 3 says "Post-multiply point with matrixObject." but what does that mean exactly? Specifically, it's unclear how to handle the w value.

Does it mean just initialize a matrix with the x, y, z, w values as:

`[1 0 0 x]`
`[0 1 0 y]`
`[0 0 1 z]`
`[0 0 w 1]`

and then post-multiply that with 'matrix'?

### PR DESCRIPTION
**Pull request by zcorpan**
Fixes #183.

### CODE REVIEW

File: geometry/Overview.bs [hunk 1]
```diff
@@ -215,6 +208,50 @@ and returns a new {{DOMPoint}} object.
...
+
+ <li><p>Let <var>w</var> be <var>point</var>'s <a for=point>w perspective</a>.
+
+ <li>
+  <p>Let <var>pointVector</var> be a new column vector with the elements being <var>x</var>,
+  <var>y</var>, <var>z</var>, and <var>w</var>, respectively.
+
+  <p><math display=inline><mrow><mfenced open=[ close=]><mrow><mtable><mtr><mtd><mi>x</mi></mtd></mtr><mtr><mtd><mi>y</mi></mtd></mtr><mtr><mtd><mi>z</mi></mtd></mtr><mtr><mtd><mi>w</mi></mtd></mtr></mtable></mrow></mfenced></mrow></math>
+
+ <li><p><a for=matrix>Post-multiply</a> <var>pointVector</var> with <var>matrix</var>.
```
Comment (fsoder) [inline geometry/Overview.bs]: Should this say something about where/how the result is written. I.e this is "matrix * pointVector", but should it be specified as "pointVector = matrix * pointVector" or "pointVector' = matrix * pointVector" or something similar for improved clarity? (The spec isn't entirely consistent in these cases it seems - in some cases the result is assigned and in some cases it isn't.)

Comment (zcorpan) [inline geometry/Overview.bs]: Yes this should be fixed (here and in general).

**Comment by zcorpan** [comment]
cc @hwanseung

File: geometry/Overview.bs (hunk 1 above)
Comment (zcorpan) [inline geometry/Overview.bs]: Fixed for this PR.

**Comment by zcorpan** [comment]
@smfr @cdumez does this PR match WebKit's implementation? https://trac.webkit.org/changeset/217763/webkit/

**Pull request merged**